src/claude_client.py

```
@@ -0,0 +1,91 @@
+            try:
+                response = client.beta.messages.create(
+                    model=_MODEL,
+                    max_tokens=_MAX_TOKENS,
+                    tools=[_WEB_SEARCH_TOOL],  # type: ignore[list-item]
+                    messages=messages,
+                    betas=["web-search-2025-03-05"],
+                )
+                break
+            except anthropic.RateLimitError as e:
+                if attempt == _RATE_LIMIT_RETRIES:
+                    raise RuntimeError(f"Anthropic レート制限（{_RATE_LIMIT_RETRIES}回リトライ後）: {e}") from e
```

Ruff (0.15.10)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)